.github/workflows/docker-image-master.yml

```diff
@@ -91,46 +91,46 @@ jobs:
-
-      - name: Run Module Unit Test
-        run: |
-          docker run build-${TARGET_VERSION}-${GITHUB_REF##*/}-${GITHUB_RUN_NUMBER} catkin_make --use-ninja run_tests
-          docker run build-${TARGET_VERSION}-${GITHUB_REF##*/}-${GITHUB_RUN_NUMBER} catkin_make --use-ninja tests
-
-      - name: Create Docker Image Tag
-        run: |
-          docker tag build-${TARGET_VERSION}-${GITHUB_REF##*/}-${GITHUB_RUN_NUMBER} ${GITHUB_REMOTE_URL}/${IMAGE_NAME}:${ARCH}-${TARGET_TYPE}-${TARGET_VERSION}
-          docker tag build-${TARGET_VERSION}-${GITHUB_REF##*/}-${GITHUB_RUN_NUMBER} ${GITHUB_REMOTE_URL}/${IMAGE_NAME}:${ARCH}-${TARGET_TYPE}-latest
-
-      - name: Push Image to Github Packages Registry
-        run: |
-          docker push --all-tags ${GITHUB_REMOTE_URL}/${IMAGE_NAME}
+  # build-ros-perception-l4t-arm64:
+  #   name: "Build ROS perception L4T ARM64 (NVIDIA XAVIER)"
+  #   runs-on: ubuntu-latest
+  #   env:
+  #     BASE_IMAGE: "docker.pkg.github.com/sonia-auv/l4t-ros-base/l4t_ros_base:arm64-perception-latest"
+  #     ARCH: arm64
+  #     TARGET_TYPE: perception-l4t
+  #     IMAGE_NAME: sonia_common
+  #     GITHUB_REMOTE_URL: docker.pkg.github.com/${{ github.repository }}
+  #   steps:
+  #     - uses: actions/checkout@v2
+  #     - name: Set Target version
+  #       run: echo "TARGET_VERSION=release-$(sed -n -e 's/.*<version>\(.*\)<\/version>.*/\1/p' package.xml)" >> $GITHUB_ENV
+
+  #     - name: Install QEMU to be able to compile on X86 into ARM64
+  #       run: |
+  #         sudo apt-get update && sudo apt-get install qemu binfmt-support qemu-user-static
+  #         docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
+
+  #     - name: Login to Github Package Registry
+  #       run: |
+  #         echo "${{ secrets.GITHUB_TOKEN }}" | docker login docker.pkg.github.com -u ${{ github.actor }} --password-stdin
+
+  #     - name: Build the docker image (perception based)
+  #       run: |
+  #         docker build . --tag build-${TARGET_VERSION}-${GITHUB_REF##*/}-${GITHUB_RUN_NUMBER} --build-arg BASE_IMAGE=${BASE_IMAGE}
+
+  #     - name: Run Module Unit Test
+  #       run: |
+  #         docker run build-${TARGET_VERSION}-${GITHUB_REF##*/}-${GITHUB_RUN_NUMBER} catkin_make --use-ninja run_tests
+  #         docker run build-${TARGET_VERSION}-${GITHUB_REF##*/}-${GITHUB_RUN_NUMBER} catkin_make --use-ninja tests
+
+  #     - name: Create Docker Image Tag
+  #       run: |
+  #         docker tag build-${TARGET_VERSION}-${GITHUB_REF##*/}-${GITHUB_RUN_NUMBER} ${GITHUB_REMOTE_URL}/${IMAGE_NAME}:${ARCH}-${TARGET_TYPE}-${TARGET_VERSION}
+  #         docker tag build-${TARGET_VERSION}-${GITHUB_REF##*/}-${GITHUB_RUN_NUMBER} ${GITHUB_REMOTE_URL}/${IMAGE_NAME}:${ARCH}-${TARGET_TYPE}-latest
+
+  #     - name: Push Image to Github Packages Registry
+  #       run: |
+  #         docker push --all-tags ${GITHUB_REMOTE_URL}/${IMAGE_NAME}
```

Comment:
This code should either be removed or uncommented.

Reply:
was kept if @0x72D0 needs it